opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -402,8 +402,7 @@ export const stixCoreObjectImportPush = async (context, user, id, file, noTrigge
     const eventFile = storeFileConverter(user, up);
     const files = [...(previous.x_opencti_files ?? []).filter((f) => f.id !== up.id), eventFile];
     await elUpdateElement({
-      _index: previous._index,
-      internal_id: internalId,
+      ...previous,
```

Comment:
this is to prevent the error that is thrown due to new check on `element.entity_type` existence in my validate function.

This makes me wonder if this was properly working in the first place ! because then it would replace in elastic with missing pieces of the object.

@richard-julien wdyt?